Source/WebKit/UIProcess/Cocoa/WKURLSchemeHandlerAdapter.swift

```diff
@@ -38,7 +38,12 @@ final class WKURLSchemeHandlerAdapter: NSObject, WKURLSchemeHandler {
     func webView(_ webView: WKWebView, start urlSchemeTask: any WKURLSchemeTask) {
         let task = Task {
             do {
-                for try await result in wrapped.reply(for: urlSchemeTask.request) {
+                // Safety:
+                // FIXME: <rdar://154775389>
+                // This is believed to be a false-positive and this code is actually safe;
+                // a future Swift compiler will remove the need for the 'unsafe' keyword
+                // here.
+                for try await unsafe result in wrapped.reply(for: urlSchemeTask.request) {
```

Comment:
Out of curiosity: With `-Werror StrictMemorySafety`, once we get a build of the compiler with this false-positive fixed, will this use site become an error?